src/plugins/janus_videoroom.c

```diff
@@ -8458,6 +8468,7 @@ static void *janus_videoroom_handler(void *data) {
 					}
 					janus_mutex_lock(&publisher->streams_mutex);
 					const char *mid = json_string_value(json_object_get(s, "mid"));
+					const char *opaqueid = json_string_value(json_object_get(s, "opaque_id"));
```

Comment:
New attributes that are added must be added to the related `_parameters` validation too.

Reply:
Am i missing something? It is added in the subscriber_stream_parameters object...